stdlib/public/core/MutableCollection.swift

```diff
@@ -173,33 +173,34 @@ where SubSequence: MutableCollection
   /// - Complexity: O(1)
   mutating func swapAt(_ i: Index, _ j: Index)
   
-  /// Call `body(p)`, where `p` is a pointer to the collection's
-  /// mutable contiguous storage.  If no such storage exists, it is
-  /// first created.  If the collection does not support an internal
-  /// representation in a form of mutable contiguous storage, `body` is not
-  /// called and `nil` is returned.
-  ///
-  /// Often, the optimizer can eliminate bounds- and uniqueness-checks
-  /// within an algorithm, but when that fails, invoking the
-  /// same algorithm on `body`\ 's argument lets you trade safety for
-  /// speed.
```

Comment:
I think we should keep these documented, even if they are deprecated

Reply:
I restored the documentation, even though they're both deprecated and internal.